public/services/requests/ppl.ts

```diff
@@ -19,11 +19,15 @@ export default class PPLService {
       query: string;
       format: string;
     },
+    dataSourceMDSId?: string,
```

Comment:
With this change do we need to add MDS param here: https://github.com/opensearch-project/dashboards-observability/blob/19d68b2d443458c3fdb1bd0fe035bf823111015b/public/services/timestamp/timestamp.ts#L81C9-L81C28

Reply:
Discussed with @sumukhswamy offline this isn't being used by metrics analytics.